test/functional/wallet_listsinceblock.py

```diff
@@ -110,7 +113,7 @@ def test_reorg(self):
         # generate on both sides
         lastblockhash = self.nodes[1].generate(6)[5]
         self.nodes[2].generate(7)
-        self.log.info('lastblockhash=%s' % (lastblockhash))
+        self.log.debug('lastblockhash=%s' % (lastblockhash))
```

Comment:
nit: While changing this line, could also switch to `''.format(...)`

Reply:
agreed